src/interpreter/mod.rs

```diff
@@ -996,13 +982,13 @@ impl<'a, 'b, 'mir, 'tcx> FnEvalContext<'a, 'b, 'mir, 'tcx> {
                 use rustc::mir::repr::Literal::*;
                 match *literal {
                     Value { ref value } => Ok(self.const_to_ptr(value)?),
-                    Item { .. } => unimplemented!(),
+                    Item { .. } => Err(EvalError::Unimplemented(format!("function pointers are unimplemented"))),
```

Comment:
I'd be happy with `"literal items (e.g. function pointers) are unimplemented"` just to avoid the possibility of future confusion.